### DISCUSSION TITLE
Is this project still alive?

### DISCUSSION THREAD

**Discussion by artokun** [liveshowy/webauthn_components#48] [General]
It's been a few months since the last commit...

**Comment by type1fool** [comment]
Hello, @artokun! Sorry I didn't see this sooner.

I have been working on some big improvements to the project over the last few weeks, primarily token improvements and code generators. [PR 50](https://github.com/liveshowy/webauthn_components/pull/50) is open, and I'm revising documentation to capture the changes. I would appreciate feedback on the PR if you are so inclined. Getting started should be much easier once this work is completed.

Thank you.

**Reply by MitchellJeppson** [reply]
Gonna piggy back on this thread. Got anything in particular I can help with? Also curious what is needed to be considered production ready!

**Reply by type1fool** [reply]
Hey @MitchellJeppson! Thanks for checking in.

The next planned enhancement would be supporting [registration for additional Passkeys](https://github.com/liveshowy/webauthn_components/issues/29) for a user. While I haven't spent much time thinking it through yet, the plan is to update `RegistrationComponent` if necessary and show how it can be used within a user profile page for adding secondary credentials. So, the work will touch both `webauth_components` and `webauthn_components_demo` repos.

To my mind, multi-key support is the only _feature_ that would prevent this from being production ready. I've been using the component in an application that has only a few dev users so far, and things are working well. However, I'd like to get more feedback from the community before releasing 1.0.

There may also be some edge cases for apps with multiple domains/origins and other config complications.

Would you like to contribute to multi-key support?

**Reply by MitchellJeppson** [reply]
Solid dude. I'll take a crack at it in my freetime and see what I can do 👍
